Update billing-understand-vm-reservation-charges.md

jboeshart opened the pull request:
Updated this document with a new visual and a different explanation of how RI's are allocated across VMs. The prior explanation didn't make it clear that they are not applied across concurrently running VMs, and I think this does a little better job. I welcome any feedback.

azurecla commented:
Hi __@thomasberman__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@thomasberman Thank you for this contribution.  I'll have the author review.

@Thraka Would you please review this change and let me know if I should merge it?  Thank you.

Thraka commented:
Looks good.

jboeshart committed:
Update billing-understand-vm-reservation-charges.md

Changed the scenario and diagram as it doesn't clearly indicate that concurrently running VMs cannot consume a single RI. I think this better explains how RI's are allocates across instances, but welcome your feedback.

jboeshart committed:
Uploaded new visual

New chart that I think better shows how RI's work across multiple VMs
